industry_restaurant/demo/product_product.xml

```diff
@@ -1,54 +1,32 @@
-    </record>
-    <record id="product_product_35" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_36" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_38" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_39" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_40" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_41" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_42" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_43" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_47" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_48" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_32" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_49" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_50" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_51" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_52" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
-    <record id="product_product_53" model="product.product">
-        <field name="is_published" eval="True"/>
-    </record>
+        <record id="product_product_54" model="product.product">
+            <field name="seller_ids" eval="[(6, 0, [ref('product_supplierinfo_1')])]"/>
+        </record>
+        <record id="product_product_55" model="product.product">
+            <field name="seller_ids" eval="[(6, 0, [ref('product_supplierinfo_2')])]"/>
+        </record>    
+        <record id="product_product_56" model="product.product">
+            <field name="seller_ids" eval="[(6, 0, [ref('product_supplierinfo_3')])]"/>
+        </record>
+        <record id="product_product_57" model="product.product">
+            <field name="seller_ids" eval="[(6, 0, [ref('product_supplierinfo_4')])]"/>
+        </record>
+        <record id="product_product_58" model="product.product">
+            <field name="seller_ids" eval="[(6, 0, [ref('product_supplierinfo_5')])]"/>
+        </record>
+        <record id="product_product_59" model="product.product">
```

Comment:
Most of its definition should be in data

Reply:
It cannot the supplier require partner.

Reply:
Yes, some parts should remain here, hence the "most of" in my comment ;-)